src/selfplay/loop.cc

```diff
@@ -75,6 +75,15 @@ void SelfPlayLoop::CmdStart() {
 }
 
 void SelfPlayLoop::SendGameInfo(const GameInfo& info) {
+  // Send separate resign report before gameready as client gameready parsing
+  // will easily get confused by adding new parameters as both training file
+  // and move list potentially contain spaces.
+  if (info.min_false_positive_threshold) {
+    std::string resign_res = "resign_report";
+    resign_res += 
+        " fp_threshold " + std::to_string(*info.min_false_positive_threshold);
+    SendResponse(resign_res);
```

Comment:
I think I should be using a SendResponses at end of function and a vector of strings instead - so the output mutex is held across both outputs to prevent interleaving.
I'll do that unless someone else has a better idea before I make the change :P